.gitignore

```diff
@@ -3,3 +3,4 @@
 build/
 accord-core/build/
 accord-maelstrom/build/
+.rat-excludes.txt
```

Comment:
nope, it's so I can build locally

Reply:
can you add a basic excludes file? Otherwise it doesn't want to build

Reply:
hmm, does it work to check-in a file and also add to .gitignore? I vaguely recall it might

Reply:
for now I've updated the build to only use it if present

Reply:
https://app.circleci.com/pipelines/github/belliottsmith/cassandra/452/workflows/866c7678-a67b-45cd-97b1-986fca944af3/jobs/7917

```
[exec]   - In plugin 'org.nosphere.apache.rat-base' type 'org.nosphere.apache.rat.RatTask' property 'excludeFile' specifies file '/home/cassandra/cassandra/modules/accord/.rat-excludes.txt' which doesn't exist.
     [exec]  
```

Reply:
> hmm, does it work to check-in a file and also add to .gitignore? I vaguely recall it might

yes, you are telling git to not add/track changes... so if you to something like `git commit -a` it won't add the changes to that file to your commit

Reply:
you can manually add the change though. This is even better though:

> for now I've updated the build to only use it if present